build/docker-compose.common.yml

```diff
@@ -16,6 +16,28 @@ services:
       interval: 5s
       timeout: 10s
       retries: 3
+    volumes:
+      - ./docker/mysql.conf.d/source.cnf:/etc/mysql/conf.d/mysql.cnf
+  mysql-read-replica:
+    image: "mysql:5.7"
```

Comment:
MySQL [support replication from 5.6 to 5.7](https://dev.mysql.com/doc/mysql-replication-excerpt/5.7/en/replication-compatibility.html).